caps/views.py

```diff
@@ -256,6 +234,128 @@ def get_context_data(self, **kwargs):
         # need to convert to a dict as items doesn't work on defaultdicts
         # in django templates
         context["grouped_documents"] = dict(grouped_documents)
+        return context
+
+    def get_council_card_context(self, context: dict[str, Any]) -> dict[str, Any]:
+        """
+        Return the council cards that are valid for this council
+        """
+
+        class MenuItem(NamedTuple):
+            """
+            Helper object to track valid council cards
+            """
+
+            slug: str
+            title: str
+            color: str = "green"
+            desc: str = ""
+
+        # You also need to add any new slugs to content-navbar.scss
+        menu = [
+            MenuItem(
+                slug="summary",
+                title="Summary",
+                desc="Overview of CAPE and information available.",
+            ),
+            MenuItem(
+                slug="new-council",
+                title="What did this council replace?",
+                desc="Details of councils this authority replaced.",
+            ),
+            MenuItem(
+                slug="old-council",
+                title="What replaced this council?",
+                desc="Details of councils that replaced this authority.",
+            ),
+            MenuItem(
+                slug="powers",
+                title="Powers & responsibilities",
+                desc="What the council is responsible for and how it relates to climate change.",
+            ),
+            MenuItem(
+                slug="declarations",
+                title="Pledges and declarations",
+                desc="Pledges and declarations the council has made around climate change.",
+            ),
+            MenuItem(slug="council-documents", title="Council documents", desc=""),
```

Comment:
Likewise, `Council documents` versus `Climate documents`. Not sure which one we want to pick here.